node/test/go/dev-mode/test/integration/index.test.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const { expect } = require('chai');
+
+const path = require('path');
+const log = require('log').get('test');
+const cleanup = require('../lib/cleanup');
+const createCoreResources = require('../lib/create-core-resources');
+const processFunction = require('../lib/process-function');
+const resolveTestVariantsConfig = require('../lib/resolve-test-variants-config');
+
+for (const name of ['TEST_EXTERNAL_LAYER_FILENAME']) {
+  // In tests, current working directory is mocked,
+  // so if relative path is provided in env var it won't be resolved properly
+  // with this patch we resolve it before cwd mocking
+  if (process.env[name]) process.env[name] = path.resolve(process.env[name]);
+}
+
+describe('integration', function () {
+  this.timeout(120000);
+  const coreConfig = {};
+
+  const useCasesConfig = new Map([
+    [
+      '4sLogger',
+      {
+        variants: new Map([
+          ['v14', { configuration: { Runtime: 'nodejs14.x', Timeout: 5 } }],
+          ['v16', { configuration: { Runtime: 'nodejs16.x', Timeout: 5 } }],
+        ]),
```

Comment:
Test seems to get back to common part

Reply:
Ahh yeah it looks like I removed that last commit I had made 🤦 I aded this back and fixed the `4sLogger` file name so it now reads `4s-logger` 👍